stable/chartmuseum/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Host your own Helm Chart Repository
 name: chartmuseum
-version: 2.10.0
+version: 2.10.1
```

We should bump the minor version, since we're adding functionality.

```suggestion
version: 2.11.0
```